### ISSUE TITLE
Replicating published results

### ISSUE THREAD

**Issue by CarlinLiao** [PathologyDataScience/MuTILs_Panoptic#5]
I'd like to replicate the results you've shown in your recent publication on PanopTILs. (Table 1 to start with but ideally every table and figure.) How would you recommend doing this?

My guess would be to start by running [`MuTILsTrainer.py`](https://github.com/PathologyDataScience/MuTILs_Panoptic/blob/main/mutils_panoptic/MuTILsTrainer.py), adjusting variables like `BASEPATH` to match the directory from which I've downloaded and unzipped PanopTILs bootstrapped set to. However, I noticed that the Trainer main calls [`utils.GeneralUtils.save_configs`, which has been commented out in the GeneralUtils file.](https://github.com/PathologyDataScience/MuTILs_Panoptic/blob/402905c6e512c4dcb231674eeaa4ab196003bf06/utils/GeneralUtils.py#L307-L324) Should I avoid using it?

Once I have a trained model, would I use `MuTILsWSIRunner` or `MuTILsAnalyticalValidation` to evaluate my trained model on the manually labeled test set across the five folds?

Thank you for your help.

**Comment by cooperlab** [comment]
We have some written and recorded documentation that I can share with you. Let me consult with the students and see if they consent to sharing the recording. Please email me at lee dot cooper at northwestern dot edu and I will find a way to get this to you.

You will not be able to replicate exactly what is shown in the tables since a portion of the training data cannot be shared. The majority of the data is accessible though.

**Comment by Dadatata-JZ** [comment]
> We have some written and recorded documentation that I can share with you. Let me consult with the students and see if they consent to sharing the recording. Please email me at lee dot cooper at northwestern dot edu and I will find a way to get this to you.
> 
> 
> 
> You will not be able to replicate exactly what is shown in the tables since a portion of the training data cannot be shared. The majority of the data is accessible though.

Can you also share the model wrights with us? Not only for MuTILs but it seems essential if we wanted to use HIPs to directly quantify slides.  

Thanks

**Comment by cooperlab** [comment]
I have hosted the original weights for download and have gathered documentation on running the model and the pipeline outputs. I'm waiting on the author to have a break so that we can update the repo and project website.

**Comment by Dadatata-JZ** [comment]
Super! Many thanks!

**Comment by jackhanyuan** [comment]
> I have hosted the original weights for download and have gathered documentation on running the model and the pipeline outputs. I'm waiting on the author to have a break so that we can update the repo and project website.
Can you share the link？  @cooperlab

**Comment by Elliot-QxZhang** [comment]
> I have hosted the original weights for download and have gathered documentation on running the model and the pipeline outputs. I'm waiting on the author to have a break so that we can update the repo and project website.

Hi, many thanks for sharing this great work. Could you pls share this link?

**Comment by cooperlab** [comment]
Model weights added in PR #8
